src/libraries/Microsoft.PowerFx.Interpreter/Functions/CommonErrors.cs

```diff
@@ -18,6 +18,16 @@ public static ErrorValue RuntimeTypeMismatch(IRContext irContext)
             });
         }
 
+        public static ErrorValue MaxCallDepth(IRContext irContext)
+        {
+            return new ErrorValue(irContext, new ExpressionError()
+            {
+                Message = "Max call depth reached",
```

Comment:
>Max call depth reached [](http://example.com/codeflow?start=27&length=22)

// TODO: Validate error message with greg

Reply:
Can be changed in another PR, just wanna get this through.